src/nodes/registry.h

```diff
@@ -23,7 +24,7 @@ class NodeRegistry
     // Check whether the supplied node type is known
     static bool hasNodeType(std::string_view nodeType);
     // Search for the supplied node type, returning strict node type if found
-    static std::string_view getNodeTypeFuzzy(std::string_view weakNodeType);
+    static std::vector<std::string_view> getNodeTypeFuzzy(std::string_view weakNodeType);
```

Comment:
Naming suggestion. Also the comment on the line above (which I can't attach a comment on) should be rewritten.
```suggestion
    static std::vector<std::string_view> getNodeTypesFuzzy(std::string_view weakNodeType);
```